src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -1041,7 +1046,10 @@ async def sync(self, job, id, options):
         Run the cloud_sync job `id`, syncing the local data to remote.
         """
 
-        cloud_sync = await self._get_instance(id)
+        cloud_sync = await self.get_instance(id)
+        if cloud_sync['task_locked']:
+            self.middleware.call_sync('alert.oneshot_create', 'CloudSyncTaskLocked', cloud_sync)
```

Comment:
I think we need to do this when generating `cron` config. Otherwise, we'll only issue alert when the task runs on schedule, which may be too late (while for share we issue it on boot)